### ISSUE TITLE
PIL is an unspecified dependency of cartopy

### ISSUE THREAD

**Issue by sgillies** [SciTools/cartopy#683]
`pip install cartopy` doesn't pull in PIL or Pillow, but it could if they were added to setup.py.

**Comment by pelson** [comment]
We've got a few layers of requirements, which you can see at https://github.com/SciTools/cartopy/tree/master/requirements. For instance, matplotlib is not a dependency on cartopy either (but if you want the matplotlib interface, you will need it).
I didn't check, but does cartopy actually import without PIL? It's hard to know what to do about this one - we could change it so that all of the optional dependencies are also installed when pip installing...

**Comment by jsegu** [comment]
I experience the same issue after installation through pip, and no, cartopy does not import without PIL:

```python
[julien@altair figures]$ python2 -c 'import cartopy'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/julien/.local/lib/python2.7/site-packages/cartopy/__init__.py", line 111, in <module>
    import cartopy.feature
  File "/home/julien/.local/lib/python2.7/site-packages/cartopy/feature.py", line 32, in <module>
    from cartopy.io.ogc_clients import WFSGeometrySource
  File "/home/julien/.local/lib/python2.7/site-packages/cartopy/io/ogc_clients.py", line 43, in <module>
    from PIL import Image
ImportError: No module named PIL
```

**Comment by QuLogic** [comment]
PIL/Pillow is used here:
```
lib/cartopy/tests/test_img_nest.py:from PIL import Image
lib/cartopy/tests/mpl/test_images.py:from PIL import Image
lib/cartopy/io/ogc_clients.py:from PIL import Image
lib/cartopy/io/img_tiles.py:from PIL import Image
lib/cartopy/io/img_nest.py:from PIL import Image
```

They are not optional in those cases, though the functionality there is not necessarily required. @pelson it's up to you whether you prefer the hard dependency or changing those files.

**Comment by pelson** [comment]
> @pelson it's up to you whether you prefer the hard dependency or changing those files.

I'm happy for you to make that call too. If I had to lean one way or the other, I'd be inclined to stop importing ```ocg_clients``` when ```cartopy``` is imported.

**Comment by QuLogic** [comment]
This should be fixed with #695.
